apps/dashboard/app/views/projects/_readme.html.erb

```diff
@@ -0,0 +1,7 @@
+<% if File.extname(@project.readme_path) == '.md' %>
```

Comment:
I think 2 things here. The first is easy, that whatever `div` or similar this is in, it should be collapsable so I'd like to add that in now.

Secondly, I'm sure there's some eaiser way in the model or similar that we can just render the `@content` (or simiarly named variable) and push all the logic elsewhere instead of having so much logic in a view.